.github/workflows/ci.yml

```diff
@@ -2,6 +2,9 @@ name: CI
 on:
   push:
     branches: [staging, trying]
+concurrency:
+    group: ci-${{ github.ref}}
+    cancel-in-progress: true
 jobs:
```

Comment:
This isn't mentioned in a commit or the PR.  Please elaborate on this is needed?  I assume its unrelated to the rest of the changes and should be its own commit or even its own PR.

Reply:
This is in there for the sake of cancelling CI runs when bors either crashes or we do `bors r-` on something that's running or on the general `trying` branch when we do `bors try`

Reply:
Could you either
1. Split this out into its own PR
2. Split this out into its own commit
3. Call this out in the PR description

(ordered by preference, reverse order for amount of work :) )